RecoMuon/MuonIdentification/test/ME0MuonAnalyzer.cc

```diff
@@ -217,98 +230,154 @@ ME0MuonAnalyzer::ME0MuonAnalyzer(const edm::ParameterSet& iConfig)
 }
 
 
+  std::cout<<"At start of begin run"<<std::endl;  
 
-//void ME0MuonAnalyzer::beginJob(const edm::EventSetup& iSetup)
-void ME0MuonAnalyzer::beginJob()
```

Comment:
You've taken out the function specifier, so everything after these lines the compiler thinks are supposed to be global statements.  I think that's the source of all the compiler errors.

Also, cout should be LogDebug or similar.